File: HackerRank/Data Structures/Linked List/removeDuplicates.c
SinglyLinkedListNode* removeDuplicates(SinglyLinkedListNode* llist) {
    SinglyLinkedListNode* current = llist;
    SinglyLinkedListNode* nextNode = current->next;
    while (current != NULL && current->next != NULL) {
        while (current->next != NULL && current->data == nextNode->data) {
            nextNode = nextNode->next;
            current->next = nextNode;
        }
        current = current->next;
        nextNode = nextNode->next;
    }
    return llist; 
}

File: LeetCode/26. Remove Duplicates from Sorted Array/RemoveDuplicates.cs
public int RemoveDuplicates(int[] nums) {
    int length = nums.Length;
    int i = 0;
    int j = 1;

    while (j < length) {
        if (nums[i] != nums[j]) {
            i++;
            nums[i] = nums[j];
        }
        j++;
    }
    return i+1;
}
